### DISCUSSION TITLE
Welcome to filester Discussions!

### DISCUSSION THREAD

**Discussion by roozbehzarei** [roozbehzarei/filester#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by katastrophe92** [comment]
Hello 👋. Where to find the deletion token? Thanks!

**Reply by roozbehzarei** [reply]
Hi,
Filester does not store deletion tokens as of yet. It will be implemented in the upcoming release.
Regards, Roozbeh.
